File: src/Traki.Domain/Models/Section/Items/Item.cs
namespace Traki.Domain.Models.Section.Items
{
    public class Item
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string? ItemImage { get; set; }
        public int Priority { get; set; }
        public Question? Question { get; set; }
        public MultipleChoice? MultipleChoice { get; set; }
        public TextInput? TextInput { get; set; }
        public int ChecklistId { get; set; }
    }
}

File: src/Traki.App/src/contracts/protocol/items/Item.ts
import { Question } from './Question';
import { MultipleChoice } from './MultipleChoice';
import { TextInput } from './TextInput';

export type Item = {
  id: string,
  name: string,
  itemImage: string|undefined,
  priority: number,
  question: Question|undefined,
  multipleChoice: MultipleChoice|undefined,
  textInput: TextInput|undefined
}